src/measure_theory/measure_space.lean

```diff
@@ -986,7 +986,8 @@ end
 
 @[simp] lemma restrict_add_restrict_compl (hs : measurable_set s) :
   μ.restrict s + μ.restrict sᶜ = μ :=
-by rw [← restrict_union disjoint_compl_right hs hs.compl, union_compl_self, restrict_univ]
+by rw [← restrict_union (@disjoint_compl_right (set α) _ _) hs hs.compl,
+    union_compl_self, restrict_univ]
```

Comment:
I'm not sure why the proof here broke; do I need to do something with the `set` instances?

Reply:
I think the explanation is in [this Zulip thread](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/add.20a.20new.20superclass.2C.20need.20some.20new.20.40s) but I'm still digesting it...